includes/definitions/pulse.yaml

```diff
@@ -10,3 +10,4 @@ discovery:
     -
         sysObjectID:
             - .1.3.6.1.4.1.12532.
+            - .1.3.6.1.4.1.8072.
```

Comment:
This would match a lot of devices, is doesnt belong to pulse secure.

Reply:
Hmm, yes I see.
This isn't as simple as I had hoped, I will cancel this PR and see if I can work out something else.